components/DropdownLight/DropdownLight.unit.test.jsx

```diff
@@ -120,4 +120,19 @@ describe('<DropdownLight />', () => {
     expect(input.value).toEqual(bananaItem);
     expect(screen.queryByRole('list')).not.toBeInTheDocument();
   });
+
+  it('should call onChange when a list item is selected using keyboard', () => {
+    const onChangeMock = jest.fn();
+    render(<DropdownLight items={itemsStringMock} onChange={onChangeMock} />);
+
+    userEvent.tab();
+    userEvent.keyboard(ENTER_KEY_CODE);
+
+    userEvent.keyboard(
+      `${ARROW_DOWN_KEY_CODE}${ARROW_DOWN_KEY_CODE}${ENTER_KEY_CODE}`,
+    );
+
+    expect(onChangeMock).toHaveBeenCalled();
```

Comment:
Is this test necessary? Apparently the test below already validates the expected behavior

Reply:
I think it's necessary, I made an adaptation using the objects to ensure that onChange is working correctly.

```suggestion
    it('should call onChange when a list item is selected using keyboard', () => {
    const onChangeMock = jest.fn();
    render(<DropdownLight items={itemsObjectMock} onChange={onChangeMock} />);

    userEvent.tab();
    userEvent.keyboard(ENTER_KEY_CODE);

    userEvent.keyboard(
      `${ARROW_DOWN_KEY_CODE}${ARROW_DOWN_KEY_CODE}${ENTER_KEY_CODE}`,
    );

    expect(onChangeMock).toHaveBeenCalled();
    expect(onChangeMock).toHaveBeenCalledWith({label: 'Strawberry', value: 'Strawberry',});
  });
```